_gsdocs-proposals/2020/proposal_ROOTdocumentation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ROOT Documentation
+layout: gsdocs_proposal
+project: ROOT
+year: 2020
+organization:
+  - CERN
+---
+
+## Description of project idea
+
+The ROOT system [1] [2] is the main data analysis and data presentation tool used in high
+energy physics world-wide. More than 1 exabyte of data are stored in the ROOT format.
+
+ROOT is being made available under the LGPL 2.1+, which allows
+ROOT to be used in a wide range of open and closed environments.
+
+ROOT it is written in C++ with dynamic Python bindings. Thanks to its
+capabilities and performance it is also used in industry.
+
+ROOT's documentation and web site are a key parts of the ROOT project [6]. They are
+evolving in parallel and complement each other. The basis is the reference Guide [4]
+documenting all the ROOT C++ components. A large set of tutorials [5], helping to
+enter the ROOT world, is also provided. And finally a "Manual"
+guides users trough the ROOT tools and concepts [7].
+
+## Tasks
+
+ Possible tasks might include:
+
+- Because ROOT's documentation is mainly physics-oriented, it is more
+  difficult for non-physicists to pick up ROOT. The Manual should address this.
+- Documentation of Python bindings in the reference guide.
+- Series of new "tutorial for modern analysis" is needed to illustrate the
+  functionalities provided by ROOT 7.
+- Technical specifications for the new data format RNTuple.
+
+## Project duration
+
+This is expected to be a long (6 months) project which will possibly continue
+after this period.
+
+## Related material
+
+  1. https://root.cern
+  2. https://github.com/root-project/
+  3. https://root.cern/guides/users-guide
+  4. https://root.cern/doc/master/
+  5. https://root.cern/doc/master/group__Tutorials.html
+  6. https://github.com/root-project/web
+  7. https://root-project.github.io/web/manual/
+
+## Expected results
+
+The deliverables of the projects will be directly visible online
+in the various area mentioned in the "Tasks" paragraph. The writers
+will directly commit his work as pull requests to the web site repository.
+
+## Experience required
+
+We are seeking technical writers with a good experience.
+Technical writing skills are mandatory.
+The following technical skills are mandatory:
```

Comment:
Besides good technical writer skills, the following additional technical skills are mandatory: